backend/crates/eiffelvis_gen/src/generator.rs

```diff
@@ -244,11 +249,8 @@ impl Iterator for Iter<'_> {
         event.meta.event_type = meta_event.name().to_string();
         event.meta.id = Uuid::from_bytes(self.rng.get_mut().gen());
 
-        // TODO: allow specifying a starting time and a time delay between events
-        event.meta.time = SystemTime::now()
-            .duration_since(UNIX_EPOCH)
-            .unwrap()
-            .as_secs();
+        self.time += self.rng.get_mut().gen_bool(1.0 / 3.0) as u64;
+        event.meta.time = self.time;
```

Comment:
this is nice but we need to keep in mind this messes up consecutive runs as this goes into the future, maybe make this togglable?

Reply:
should probably just leave this out for this PR and come back to it later